pages/blog/_slug.vue

```diff
@@ -36,3 +36,15 @@ export default {
   },
 };
 </script>
+
+<style>
```

Comment:
Something funky is happening with the switch to hamburger menu. Did this ever happen before? I'll investigate but you might immediately know what's causing this.


https://github.com/brown-ccv/ccv-website-nuxt/assets/57297033/e0501b14-1ad7-419c-a092-b4b9f2612fd9

Reply:
which browser are you using? i'm not seeing this behavior on chrome